string-plugin/src/test/java/com/github/underscore/string/StringTest.java

```diff
@@ -2006,8 +2007,12 @@ public void chain() {
             public boolean test(String str) { return true; } });
         $.chain(new String[] {""}).reduce(new BiFunction<String, String, String>() {
             public String apply(String accum, String str) { return null; } }, "");
+        $.chain(new String[] {""}).reduce(new BinaryOperator<String>() {
+            public String apply(String accum, String str) { return null; } });
         $.chain(new String[] {""}).reduceRight(new BiFunction<String, String, String>() {
             public String apply(String accum, String str) { return null; } }, "");
+        $.chain(new String[] {""}).reduceRight(new BinaryOperator<String>() {
```

Comment:
\[FindBugs\] INFO: RV: Return value of $$Chain\.reduceRight\(BinaryOperator\) ignored, is this OK in com\.github\.underscore\.string\.StringTest\.chain\(\)